packages/esm-patient-list-app/src/patient-table/overflow-menu.component.tsx

```diff
@@ -0,0 +1,59 @@
+import { isDesktop, showToast, useLayoutType } from '@openmrs/esm-framework';
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { removePatientFromList } from '../api/api-remote';
+import { OverflowMenu, TableCell, Layer, OverflowMenuItem } from '@carbon/react';
+import styles from './patient-table.scss';
+
+interface OverflowMenuComponentProps {
+  cohortMembershipUuid: string;
+}
+
+const OverflowMenuComponent: React.FC<OverflowMenuComponentProps> = ({ cohortMembershipUuid }) => {
+  const { t } = useTranslation();
+  const layoutType = useLayoutType();
+  const desktopLayout = isDesktop(layoutType);
+
+  // eslint-disable-next-line no-console
+  console.log(cohortMembershipUuid);
```

Comment:
If you're logging for debugging purposes, it's probably better to put this in the callback than here.